### PR TITLE
Feature/53585 2

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: src/molecules/cookie-message/CookieMessage.stories.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      linkText: {
+        default: text('Link text', 'Example link')
+      },
+      infoText: {
+        default: text('Info text', 'Find out more about their purpose and settings in your browser. By browsing the site you are agreeing to use cookies according to your browser settings.')
+      }
+    },
+    template: `
+      <a-cookie-message
+        :visible="isVisible"
```
Comment (robwozniak) [inline src/molecules/cookie-message/CookieMessage.stories.js]: `visible` prop is `true` by default, so I am not sure if we need this line here and passing data with `isVisible` to component

File: src/molecules/alert/Alert.js [hunk 2]
```diff
@@ -2,10 +2,16 @@
 
 export default {
   props: {
+    /**
+     * Visibility status
+     */
     visible: {
```
[the remark this answers is not in the document]
Comment (robwozniak) [reply src/molecules/alert/Alert.js]: also we can remove `isVisible` data from `Alert.stories.js` where redundant (`true`) value is passing

File: src/molecules/cookie-message/CookieMessage.stories.js (hunk 1 above)
Comment (szafran89) [reply src/molecules/cookie-message/CookieMessage.stories.js]: IMO we need this b/c we change this value after close action

```
  methods: {
    close () {
      this.isVisible = false
    }
  }
```

File: src/molecules/alert/Alert.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (szafran89) [reply src/molecules/alert/Alert.js]: IMO we need this b/c we change this value after close action

```
  methods: {
    close () {
      this.isVisible = false
    }
  }
```

File: src/molecules/cookie-message/CookieMessage.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+// @vue/component
+export default {
+  props: {
+    /**
+     * Visibility status
+     */
+    visible: {
```
[the remark this answers is not in the document]
Comment (szafran89) [reply src/molecules/cookie-message/CookieMessage.js]: I would like to keep one word, maybe `visibility` is better for you?

File: src/molecules/cookie-message/CookieMessage.scss [hunk 4]
```diff
@@ -1,31 +1,33 @@
...
 $cookie-message__font-size             : $font-size-small !default;
-$cookie-message__font-family           : $font-family-base !default;
-$cookie-message__text-color            : $gray !default;
+
 $cookie-message__text-padding          : $spacer--medium $spacer--medium 0 $spacer--medium !default;
-$cookie-message__text-padding--small   : $spacer--medium $spacer $spacer--medium $spacer--medium !default;
-$cookie-message__link-color            : $blue !default;
-$cookie-message__close-color           : $cookie-message__text-color !default;
+$cookie-message__text-padding\@small   : $spacer--medium $spacer $spacer--medium $spacer--medium !default;
+$cookie-message__text-color            : $gray-dark !default;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply src/molecules/cookie-message/CookieMessage.scss]: there is wrong color on design, we have to use this one b/c of a11y purpose

File: src/molecules/cookie-message/CookieMessage.scss [hunk 5]
```diff
@@ -71,41 +67,36 @@ $cookie-message__close-icon-transition : $transition-base !default;
...
   }
 
   &__close-icon {
     width: $cookie-message__close-icon-size;
     height: $cookie-message__close-icon-size;
     margin-left: $cookie-message__close-icon-margin-left;
+    transition: $cookie-message__close-icon-transition;
     fill: $cookie-message__close-color;
 
     &:hover,
```
[the remark this answers is not in the document]
Comment (szafran89) [reply src/molecules/cookie-message/CookieMessage.scss]: that's right, removed. It was copy/past from Asia's PR

**Pull request merged**